launch/tier4_planning_launch/launch/scenario_planning/lane_driving.launch.xml

```diff
@@ -6,12 +6,10 @@
     <group>
       <push-ros-namespace namespace="behavior_planning"/>
       <group>
-        <include file="$(find-pkg-share tier4_planning_launch)/launch/scenario_planning/lane_driving/behavior_planning/behavior_planning.launch.py">
-          <arg name="vehicle_param_file" value="$(var vehicle_param_file)"/>
-          <arg name="use_multithread" value="true"/>
-          <arg name="use_pointcloud_container" value="$(var use_pointcloud_container)"/>
-          <arg name="container_name" value="$(var pointcloud_container_name)"/>
-          <arg name="use_experimental_lane_change_function" value="$(var use_experimental_lane_change_function)"/>
+        <include file="$(find-pkg-share tier4_planning_launch)/launch/scenario_planning/lane_driving/behavior_planning/behavior_planning.launch.xml">
+          <arg name="container_type" value="component_container_mt"/>
+          <!-- This condition should be true if run_out module is enabled and its detection method is Points -->
+          <arg name="launch_compare_map_pipeline" value="false"/>
```

Comment:
Should `launch_compare_map_pipeline` be changed manually by users?
In `behavior_planning_launch.py`, this condition was automatically set as true when (run_out module is launched && run_out module's method is Points).
https://github.com/autowarefoundation/autoware.universe/blob/a2b114da5f16d097b45128e29a4f94a325967030/launch/tier4_planning_launch/launch/scenario_planning/lane_driving/behavior_planning/behavior_planning.launch.py#L228-L229


If it is difficult to do the same thing in launch.xml, I think it is okay to write the comments like this and have the user set it as true.
` This option should be true if you launch the run_out module with Points method.`

Reply:
I confirmed run_out with `Points` method works when `launch_compare_map_pipeline` is true.
![image](https://github.com/autowarefoundation/autoware.universe/assets/11865769/d2969ffc-b14b-4886-8fe4-9fea7e7b3995)

Reply:
@TomohitoAndo Thanks! 

> If it's challenging to replicate this automatic behavior in launch.xml, it might be acceptable to require users to manually set it to true.

I'll try it. But it might be difficult...

Reply:
@TomohitoAndo Sorry, it is a little bit difficult to improve it quickly...
Anyway, I would like to refactor launch files so that we can launch only modules we want to use more easily. So, I'll take this issue into consideration and improve as much as I can.

Then, I would like you to change this arg manually for a while. Is it ok?

Reply:
Okay! Thank you for trying!